frontend/src/App.tsx

```diff
@@ -25,7 +24,7 @@ export default function App() {
         <BrowserRouter>
             <AcmHeader route={route}>
                 <LoadData>
-                    <AppContextContainer>
+                    <BrowserRouter>
```

Comment:
BrowserRouter twice here, look above...
BrowserRouter needed to be above AcmHeader